tests/behat/features/cms_support.feature

```diff
@@ -1,11 +1,11 @@
 @tide @install:toolbar
-Feature: CMS Support
+feature: cms support
 
   @api
-  Scenario Outline: User logs into the CMS and clicks on the Manage tab
-    Given I am logged in as a user with the "<role>" role
-    Then I should find menu item text matching "CMS Support"
-    Examples:
+  scenario outline: user logs into the cms and clicks on the manage tab
```

Comment:
Did some filter go over this to make it all lowercase? Looks weird :)

Reply:
yeah, not sure why seems like an accidental commit. deleted.